.github/workflows/labeler.yml

```diff
@@ -0,0 +1,24 @@
+name: "rails-bot: Label PRs"
+on:
+  # This event runs in the context of the base of the pull request, rather than
+  # in the context of the merge commit, as the pull_request event does. This
+  # prevents execution of unsafe code from the head of the pull request that
+  # could alter your repository or steal any secrets you use in your workflow.
+  # This event allows your workflow to do things like label or comment on pull
+  # requests from forks. Avoid using this event if you need to build or run
+  # code from the pull request.
+  #
+  # https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target
+  pull_request_target:
```

Comment:
> When a workflow is triggered by the [pull_request_target](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) event, the GITHUB_TOKEN is granted read/write repository permission, even when it is triggered from a public fork. For more information, see "[Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target)."

Reply:
> Warning: For workflows that are triggered by the pull_request_target event, the GITHUB_TOKEN is granted read/write repository permission unless the permissions key is specified and the workflow can access secrets, even when it is triggered from a fork. Although the workflow runs in the context of the base of the pull request, you should make sure that you do not check out, build, or run untrusted code from the pull request with this event. Additionally, any caches share the same scope as the base branch.

We are not checking out or running untrusted code in this workflow.

Reply:
I think we're pretty well protected here by using `pull_request_target` but would appreciate a second look. :bow: